core/modules/layout/layout.module

```diff
@@ -1741,7 +1741,7 @@ function layout_get_title_description(Layout $layout) {
     if (!empty($layout->settings['title_block'])) {
       $uuid = $layout->settings['title_block'];
       $title_block = $layout->content[$uuid];
-      $description .= ' <em>' . $all_blocks[$title_block->module][$title_block->delta]['info'] . '</em>';
+      $description .= ' <em>' . $all_blocks[$title_block->module][$title_block->delta]['info'] . '</em>' . ' block';
```

Comment:
Concatenating strings like this makes for difficult translation. Actually it looks like " block" isn't translated at all with this approach. It would be preferable to put the entire string inside a `t()` call so that the words can be rearranged as necessary within the sentence.